.github/workflows/dogfood.yaml

```diff
@@ -49,6 +49,8 @@ jobs:
           cache-from: type=registry,ref=codercom/oss-dogfood:latest
           cache-to: type=inline
   deploy_template:
+    needs: deploy_image
+    if: ${{ needs.deploy_image.result == 'success' || needs.deploy_image.result == 'skipped' }}
```

Comment:
In what situations does the job end up being skipped?

Reply:
hmm, when there are no changes in the dogfood directory. but wait that will skip the whole workflow. Good catch.

Reply:
Should we run this on success only or also on failures?

Reply:
Success only should be fine, I think. Not a big deal to change this later if it behaves differently from what we want 👍